index.js

```diff
@@ -144,12 +148,16 @@ function mirror (src, dst, opts, cb) {
 
     var rs = a.fs.createReadStream(a.name)
     var ws = b.fs.createWriteStream(b.name, {mode: a.stat.mode})
+    var increment = through(function (chunk, enc, cb) {
+      progress.emit('chunk', chunk)
```

Comment:
i think we should call this `put-data` instead of chunk. then i can add a `put-end` event as well